templates/docker-compose/Makefile.dist

```diff
@@ -83,10 +83,8 @@ upgrade: ## Upgrade SupportPal to a later version.
 	@sleep 10
 	$(DOCKER_COMPOSE_BIN) $(COMPOSE_FILES) down -v
 	$(GIT_BIN) pull
-	$(MAKE) create_volumes
-	$(DOCKER_COMPOSE_BIN) $(COMPOSE_FILES) up -d supportpal
-	$(DOCKER_BIN) exec -it $(WEB_SERVICE_NAME) bash /scripts/upgrade-helpdesk.sh
-	$(MAKE) start
+	MAKE="$(MAKE)" DOCKER_COMPOSE_BIN="$(DOCKER_COMPOSE_BIN)" COMPOSE_FILES="$(COMPOSE_FILES)" DOCKER_BIN="$(DOCKER_BIN)" WEB_SERVICE_NAME="$(WEB_SERVICE_NAME)" bash upgrade.sh
```

Comment:
This only fixes it for future upgrades. If you upgrade from 4.2.5 to 5.0.1, it still errors because the make file is evaluated based on the old contents. I think the only way we can fix that is by documenting in `MAINTENANCE.md` to run `git pull && make upgrade` ?? Not sure if that would cause a problem with the `down -v` command?

Reply:
Added note to MAINTENANCE.md